dashboard/src/actions/apps.ts

```diff
@@ -46,12 +60,74 @@ export type AppsAction = ActionType<typeof allActions[number]>;
 export function getApp(
   releaseName: string,
   namespace: string,
-): ThunkAction<Promise<void>, IStoreState, null, AppsAction> {
+): ThunkAction<Promise<hapi.release.Release | undefined>, IStoreState, null, AppsAction> {
   return async dispatch => {
     dispatch(requestApps());
     try {
       const app = await App.getRelease(namespace, releaseName);
       dispatch(selectApp(app));
+      return app;
+    } catch (e) {
+      dispatch(errorApps(e));
+      return;
```

Comment:
is this return necessary?

Reply:
It's necessary to avoid a linter error: `Not all the paths return a value`